cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/rest/CloudControllerRestClientImpl.java

```diff
@@ -884,32 +884,28 @@ public CloudServiceBroker getServiceBroker(String name, boolean required) {
         return serviceInstancesMetadata;
     }
 
-    private <T> List<List<T>> toBatches(List<T> largeList, int maxCharLength) {
+    private <T> List<List<T>> toBatches(Collection<T> largeList, int maxCharLength) {
         if (largeList.isEmpty()) {
             return Collections.emptyList();
         }
+
         List<List<T>> batches = new ArrayList<>();
+        int currentFence = 0, currentIndex = 0;
```

Comment:
I prefer the names `currentBatchLength` over `currentFence` and `currentBatchIndex` over `currentIndex`.

Reply:
Done